pages/WhatIsOppia.html

```diff
@@ -8,7 +8,7 @@
 
   <p>
     Oppia is an online learning tool that enables anyone to easily create and
-    share interactive activities ('explorations') which try to recreate the
+    share interactive activities ("explorations") which try to recreate the
```

Comment:
This is a term of art, so per our discussion, should it be single quotes instead?